apps/identity_web/lib/hydra.ex

```diff
@@ -0,0 +1,372 @@
+defmodule ORY.Hydra do
+  @moduledoc """
+  Temporary fix to get the new Hydra 2 API working with this library.
+  Can be deleted as soon as the official library is updated :
+  See : https://github.com/malomohq/ory-hydra-elixir/issues/51
+  """
+
+  alias ORY.Hydra.{Config, Operation, Request, Response}
+
+  @type http_headers_t :: [{String.t(), String.t()}]
+
+  @type http_method_t :: :delete | :get | :post | :put
+
+  @type response_t :: {:ok, Response.t()} | {:error, Response.t() | any}
+
+  @doc """
+  Accept a consent request.
+  """
+  @spec accept_consent_request(map) :: Operation.t()
+  def accept_consent_request(params) do
+    %Operation{
+      method: :put,
+      params: params,
+      params_in_query: [:consent_challenge],
+      path: "/admin/oauth2/auth/requests/consent/accept"
+    }
+  end
+
+  @doc """
+  Accept a login request.
+  """
+  @spec accept_login_request(map) :: Operation.t()
+  def accept_login_request(params) do
+    %Operation{
+      method: :put,
+      params: params,
+      params_in_query: [:login_challenge],
+      path: "/admin/oauth2/auth/requests/login/accept"
+    }
+  end
+
+  @doc """
+  Accepts a logout request.
+  """
+  @spec accept_logout_request(map) :: Operation.t()
+  def accept_logout_request(params) do
+    %Operation{
+      method: :put,
+      params: params,
+      params_in_query: [:logout_challenge],
+      path: "/admin/oauth2/auth/requests/logout/accept"
+    }
+  end
+
+  @doc """
+  Check whether an ORY Hydra service is alive.
+  """
+  @spec check_alive :: Operation.t()
+  def check_alive do
+    %Operation{
+      method: :get,
+      path: "/health/alive"
```

Comment:
I think that only health is needed here

```suggestion
      path: "/health"
```